<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommonInterfaceSelector</class>
 <widget class="QDialog" name="CommonInterfaceSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="interfaceSelectorTabs">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>401</width>
     <height>331</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="serialSelectorTab">
    <attribute name="title">
     <string>Serial</string>
    </attribute>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>381</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="serialPortContainer">
      <item>
       <widget class="QLabel" name="serialPortLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Device</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="serialPortCombo">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>381</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="serialBaudContainer">
      <item>
       <widget class="QLabel" name="serialBaudLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Baud Rate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="serialBaudCombo"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>381</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="serialButtonContainer">
      <item>
       <spacer name="leftButtonSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="serialButtonSet">
        <property name="text">
         <string>Set</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="serialButtonCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="rightButtonSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>381</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="serialDataContainer">
      <item>
       <widget class="QLabel" name="serialDataLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Data Bits</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="serialDataCombo">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>381</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="serialStopContainer">
      <item>
       <widget class="QLabel" name="serialStopLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Stop Bits</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="serialStopCombo">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>381</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="serialParityContainer">
      <item>
       <widget class="QLabel" name="serialParityLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Parity</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="serialParityCombo">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>381</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="serialFlowContainer">
      <item>
       <widget class="QLabel" name="serialFlowLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Flow Control</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="serialFlowCombo">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <zorder>label</zorder>
    <zorder>horizontalLayoutWidget_2</zorder>
    <zorder>horizontalLayoutWidget_3</zorder>
    <zorder>horizontalLayoutWidget_6</zorder>
    <zorder>horizontalLayoutWidget_7</zorder>
    <zorder>horizontalLayoutWidget_8</zorder>
    <zorder>horizontalLayoutWidget_9</zorder>
    <zorder>horizontalLayoutWidget_10</zorder>
   </widget>
   <widget class="QWidget" name="demoSelectorTab">
    <attribute name="title">
     <string>Demo</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
